concepts/language-model.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model takes text input (usually a history) and outputs the next word or character.
+
+Example:
+- input: `the man is riding a ___`
+- output: `bike` 40%, `car`: 30%, `scooter`: 10%, `blue`: 0.5%, ...
+
+In the example this would mean that the article `bike` has a 40% probability of being the continuation and is the most likely.
+
+It is possible to do *decoding*, which means repeatedly taking e.g. the most probable output, adding it to the text and so on.
```

Comment:
Maybe this deserves it's own example?

> ## Decoding
> ...
> 
> Example:
> > - input: `the man is riding a ___`
> > - output: `bike`, ...
>
> > - input: `the man is riding a bike ___`
> > - output: `to`, ...
>
> > - input: `the man is riding a bike to ___`
> > - output: `the`, ...
>
> When an end token is reached, the decoder stops.
>
> > - input: `the man is riding a bike to the store . ___`
> > - output: `<END>`, ...

Reply:
Added an example and a note on decoding.